### ISSUE TITLE
Serious bug again with MODAL preventing users to navigate

### ISSUE THREAD

**Issue by batata004** [Semantic-Org/Semantic-UI#6623]
This issue on some versions appear, on others disappear... but now it appeared again and this time it got even worse.

The test case is pretty simple. Open this -> http://quemfazsite.com.br/temp/teste.php

After you open the link above a modal will show up. Click on the blue bottom on the bottom/right. It will simply show a previously hidden element. The modal does not recalculate it's content height and thus the scroll bars of the page never show up.

This bug happens everytime you open a small modal that fits on the screen without needing scrollbars and then you make some information visible that does not fit the screen anymore. This bug is even worse than I saw on other versions cause this time the closable button at the top/right disappears.

I updated semantic ui on hundreds of domains on last month just to discover this nasty bug haunting again - but this time it got much worse.

**Comment by ColinFrick** [comment]
Agreed, you could if applicable call 'refresh' on the modal after the animation finished:
```javascript
$("#janela_entrar_contato").find(".ui.error").slideDown(1000, function() {$("#janela_entrar_contato").modal("refresh")});
```
or add the `scrolling` class to the content:
```html
<div class="scrolling content">
```

**Comment by batata004** [comment]
@ColinFrick yeap, using `refresh` used to solve the bug on older versions - in case this base happened. But I have too many websites that are using semantic ui, it would be completely impractical checking all the codes and add a `refresh` call after any content gets inserted into the modal or after any element gets visible/displayed.

It would be nice to have this bug fixed permantently so it could not become active again in the future. With a little work I can update again the semantic ui version on hundreds of websites using some regexes, but I cant use the `refresh` method - it would be too much consuming and this bug would be much better handled directly in the library instead with additional code.
